database/exip.sql

```diff
@@ -347,6 +347,7 @@ CREATE TABLE IF NOT EXISTS `PolicyAndExport` (
   `creditPeriodWithBuyer` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
 	`totalMonthsOfCover` int DEFAULT NULL,
 	`totalSalesToBuyer` int DEFAULT NULL,
+	`maximumBuyerWillOwe` int DEFAULT NULL,
```

Comment:
how does maximumBuyerOwes sound to you?

Reply:
I think the "will" part is essential - without it, it could imply that it is a definitive "the buyer owes X", but they might not actually owe it _yet_, so I think we need the "will" part?